test/test_images/probe_helper/probe_helper.go

```diff
@@ -162,6 +172,16 @@ func receiveEvent(receivedEvents *receivedEventsMap) cloudEventsFunc {
 	}
 }
 
+func healthChecker(w nethttp.ResponseWriter, r *nethttp.Request) {
+	now := time.Now()
+	if (!lastSenderEventTimestamp.IsZero() && now.Sub(lastSenderEventTimestamp) > maxStaleTime) ||
```

Comment:
The IsZero check doesn't cover the failure scenario where no events were ever received.

I think the timestamps should be initialized when starting the probeHelper, and we don't need the IsZero check.

Reply:
Makes sense. Fixed.